main.py

```
@@ -1,243 +1,714 @@
+                logging.error(f"Unexpected command error: {str(error)}", exc_info=error)
+                
+                # Send a generic error message to the user
+                await ctx.send(self.translations["error_unexpected"])
+            
+        except Exception as e:
+            # Log any errors that occur during error handling
+            logging.error(f"Error in error handler: {str(e)}", exc_info=e)
+            try:
+                await ctx.send(self.translations["error_handler_failed"])
+            except:
+                pass  # If we can't send a message, there's nothing more we can do
```

Ruff

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

(SIM105)